nodes/explorer/src/api_backend/mod.rs

```diff
@@ -84,7 +84,8 @@ where
             )
             .layer(TraceLayer::new_for_http())
             // .merge(SwaggerUi::new("/swagger-ui").url("/api-docs/openapi.json", ApiDoc::openapi()))
-            .route("/blocks", routing::get(store::store_blocks::<T, S>))
+            .route("/store/blocks", routing::get(store::store_blocks::<T, S>))
```

Comment:
`store` in the block explorer context feels not require. Both endpoints return blocks, just with a different filter. 
I'd suggest to unify both endpoint, adding ability to filter by `BlockID` + depth, and batch block retrieval by providing the `Vec<BlockID>` (do we really need batch retrieval?).

Reply:
Yeah, batch retrieval was added by `block-explorer` PR. I will remove it in another PR if we do not want it.

Reply:
Now that the `/explorer/blocks` is added, the `/store/blocks` could be removed (does the same thing).